src/traits.d

```diff
@@ -522,6 +546,15 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
     {
         return isFuncX(f => !f.needThis() && !f.isNested());
     }
+    if (e.ident == Id.isModule)
+    {
+        //TODO: can we handle the case when isPkgMod == PKGunknown?
```

Comment:
That may be true. The tricky bit is that PKGunknown means we can't tell yet whether it's a package or not, right? Or does it just mean that there's a package.d which we haven't imported?